Interfaces/About.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../Style/main.css">
+    <link rel="stylesheet" href="../Style/homePageStyle.css">
+    <link rel="stylesheet" href="../Style/About.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"> 
+    <title>About </title>
+</head>
+<body>
+  <div class="center">
+    <span></span><span></span><span></span>
+  </div>
+    <header class="header-container">
+        <div class="circle"></div>
+        <nav>
+          <ul class="first-navbar">
+            <li><i class="bi bi-facebook"></i></li>
+            <li><i class="bi bi-twitter"></i></li>
+            <li><a>English</a></li>
+            <li><a>My account</a></li>
+            <li><a>logout</a></li>
+          </ul>
+        </nav>
+          <nav class="navbar">
+            <a href="../Index.html"><img class="brand-logo" src="../Images/logo_image/logo-image.png" alt="Brand-name-and-logo"></a>
+           <ul class="nav-list-items">
+            <li class="list-item"><a  href="../Index.html">Home</a></li>
+            <li class="list-item"><a  href="../Interfaces/About.html">About</a></li>
+            <li class="list-item"><a  href="#">Program</a></li>
+            <li class="list-item"><a  href="#">Sponsor</a></li>
+            <li class="list-item"><a  href="#">New</a></li>
+            <li class="list-item"><a  href="#">Ticket</a></li>
+           </ul>
+           <div class="hambuger-container ">
+           <div class="bar"></div>
+           <div class="bar"></div>
+           <div class="bar"></div>
+           </div>
+          </nav>
+        </header>
+        <section class="landing-section1">
+          <div class="intro-section1">
+            <h1 class="openingText1">
+              Quarter mboko rappers           
+            </h1>
+            
+                 <h2 class="captureText1">
+                  About Mboko gods among us 2022
+                 </h2>
+            
+  
+                 <p class="explaining-text1">
+                    Mboko drill unearths Cameroon's hidden artists. It is a place where talent is discovered as others come to do what they do best in their careers. It began in Bamenda in 2015 and has since grown to become both national and international..
+                 </p>
+  
+                 <p class="eventDate1">Please contact us via email for any further question about mboko drill 2022</p>
+                 <span class="venue1">mboko@drill.com</span>
```

Comment:
I kindly suggest that when you post an email, you also use an "To" link to that email with the href="mailto:mboko@drill.com" attribute.